hbp_knowledge/tau/wang2015.bel

```diff
@@ -56,205 +55,275 @@ DEFINE ANNOTATION Species       AS PATTERN "^\d+$"
 DEFINE ANNOTATION Confidence    AS URL "https://arty.scai.fraunhofer.de/artifactory/bel/annotation/confidence/confidence-1.0.0.belanno"
-DEFINE ANNOTATION Disease       AS URL "https://arty.scai.fraunhofer.de/artifactory/bel/annotation/disease/disease-20170511.belanno"
-DEFINE ANNOTATION Gender        AS URL "https://arty.scai.fraunhofer.de/artifactory/bel/annotation/gender/gender-1.0.0.belanno"
+# DEFINE ANNOTATION Disease       AS URL "https://arty.scai.fraunhofer.de/artifactory/bel/annotation/disease/disease-20170511.belanno"
+# DEFINE ANNOTATION Gender        AS URL "https://arty.scai.fraunhofer.de/artifactory/bel/annotation/gender/gender-1.0.0.belanno"
 DEFINE ANNOTATION MeSHAnatomy   AS URL "https://arty.scai.fraunhofer.de/artifactory/bel/annotation/mesh-anatomy/mesh-anatomy-20170511.belanno"
 DEFINE ANNOTATION MeSHDisease   AS URL "https://arty.scai.fraunhofer.de/artifactory/bel/annotation/mesh-diseases/mesh-diseases-20170511.belanno"
-DEFINE ANNOTATION Subgraph      AS URL "https://arty.scai.fraunhofer.de/artifactory/bel/annotation/neurommsig/neurommsig-1.0.3.belanno"
-DEFINE ANNOTATION TextLocation  AS URL "https://arty.scai.fraunhofer.de/artifactory/bel/annotation/text-location/text-location-1.0.1.belanno"
+# DEFINE ANNOTATION Subgraph      AS URL "https://arty.scai.fraunhofer.de/artifactory/bel/annotation/neurommsig/neurommsig-1.0.3.belanno"
+# DEFINE ANNOTATION TextLocation  AS URL "https://arty.scai.fraunhofer.de/artifactory/bel/annotation/text-location/text-location-1.0.1.belanno"
 
 ##############
 # Statements #
 ##############
 
-
 # Add statements below this comment
 
 SET Citation = {"PubMed", "26631930"}
 
 SET Evidence = "Tauopathies can be classified into three groups on the basis of the tau isoforms found in the aggregates: 4R
 tauopathies (including PSP, CBD and AGD), 3R tauopathies (for example, PiD) and 3R+4R tauopathies (for example, AD)"
+SET Confidence = "Medium"
+path(MESH:"Supranuclear Palsy, Progressive") isA path(MESH:"Tauopathies")
+path(HBP:"Corticobasal Degeneration") isA path(MESH:"Tauopathies")
+path(MESH:"Niemann-Pick Diseases") isA path(MESH:"Tauopathies")
+path(MESH:"Alzheimer Disease") isA path(MESH:"Tauopathies")
 path(MESH:"Supranuclear Palsy, Progressive") pos p(HBP:"4R tau")
 path(HBP:"Corticobasal Degeneration") pos p(HBP:"4R tau")
 path(MESH:"Niemann-Pick Diseases") pos p(HBP:"3R tau")
 path(MESH:"Alzheimer Disease") pos p(HBP:"4R tau")
 path(MESH:"Alzheimer Disease") pos p(HBP:"3R tau")
+UNSET Confidence
 
 SET Evidence = "Indeed, knockdown of FUS increases the expression of the 2N and 4R tau isoforms, providing
 a possible link between frontotemporal lobar degeneration-FUS (FTLDFUS; a subtype of FTD characterized by FUS inclusions
 in neurons and glia) and FTLD-tau (a tauopathy)."
+SET Confidence = "High"
 p(HGNC:FUS) -| p(HBP:"4R tau")
 p(HGNC:FUS) -| p(HBP:"Tau isoform F (441 aa)")
```

Comment:
is this necessary? Since we already encode this information in the HBP ontology it automatically gets added